code/game/machinery/adv_med.dm

```diff
@@ -93,7 +93,7 @@
 		qdel(G)
 
 
-/obj/machinery/bodyscanner/MouseDrop_T(mob/living/carbon/O, mob/user as mob)
+/obj/machinery/bodyscanner/MouseDrop_T(mob/living/carbon/human/O, mob/user as mob)
```

Comment:
In retrospect, allowing the sleeper to have simple_animals in it was kind of a silly feature for me to bend over backwards for